linode/manager
# refactor: [M3-6318] - MUI v5 - Components > DeletionDialog

## Description 📝
Refactors `src/components/DeletionDialog/DeletionDialog.tsx` to remove JSS

## Major Changes 🔄
- Updated references for named export

## Preview 📷
| Before  | After   |
| ------- | ------- |
| ![Screen Shot 2023-04-24 at 11 28 54 AM](https://user-images.githubusercontent.com/125309814/234044332-188a3214-8f02-4207-ab76-96b01f5ef6ee.png) | ![Screen Shot 2023-04-24 at 11 29 07 AM](https://user-images.githubusercontent.com/125309814/234044363-7ec8bd3e-25ed-4210-9511-12d3ebe9003f.png) |

## How to test 🧪
1. `yarn dev`
2. Go to http://localhost:3000/linodes
3. Attempt to delete any linode and observe no differences
4. N/A
